wordpress-mobile/WordPress-iOS
# Disable Local Notifications and Flow for Blogging Reminders

This PR disables local notifications and flow prompts for blogging reminders based on the blogging reminders FeatureFlag.

Ref #19560

To test:
Setup
1. Enable FeatureFlag allowDisablingWPNotifications
2. Test on Device and run up WPiOS app -> Notifications -> Notification Settings and ensure the Allow Notifications switch at top is set to off

3. UI Notification Settings Test
4. Continuing your test on device in Notification Settings below the Allow Notifications UISwitch tap into one of your sites
2. Click on one of your Sites -> Push Notifications
3. Blogging Reminders should no longer be a row (you can compare to production AppStore app where Blogging Reminders is the last row)

Creating a new post
5. Create a new blog post and publish the post
6. Blogging reminder should not appear

Publish a draft post
7. Create a new blog post and save as a draft
8. Go to the main site page and choose Posts to view all the posts for the site of choice
9. Choose the Drafts tab.  Find your draft post.  Tap the … More button and tap Publish Now to publish the draft post
10. Blogging reminder should not appear

Site Stats
11. Uninstall / re-install the WP iOS app - this isn't necessary but I'd recommend in case you previously dismissed a Grow Audience card
12. Tap through to Stats for a site with hardly any traffic.  The top card that will be displayed Grow your audience type card.  Make sure that this card is not a Blogging Reminders card.  Tap dismiss (screenshot below of what the Grow Audience card looks like and Dismiss button)
13. Navigate back to the main site page and back to Stats 3 or 4 times and each time dismiss the Grow Audience card and ensure the blogging reminders card does not appear

<img width="222" alt="image" src="https://user-images.githubusercontent.com/88816724/202230551-0b030e2f-c690-40f1-b5cb-81e6a32b111d.png">



## Regression Notes
1. Potential unintended areas of impact
Blogging Reminders

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Tested on device and added a unit test

3. What automated tests I added (or what prevented me from doing so)
I added a Unit test

PR submission checklist:
